streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListener.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.streaming.scheduler
 
+import org.apache.spark.scheduler.StatsReportListener
```

Comment:
You have not addressed this. Why was this needed?? I dont see anything that required this.